source/jormungandr/jormungandr/scenarios/utils.py

```diff
@@ -280,13 +280,13 @@ def change_ids(new_journeys, journey_count):
         for i in range(len(ticket.section_id)):
             ticket.section_id[i] = ticket.section_id[i] + '_' + six.text_type(journey_count)
 
-    for new_journey in new_journeys.journeys:
+    for count, new_journey in enumerate(new_journeys.journeys):
         for i in range(len(new_journey.fare.ticket_id)):
             new_journey.fare.ticket_id[i] = new_journey.fare.ticket_id[i] \
-                                            + '_' + six.text_type(journey_count)
+                                            + '_' + six.text_type(journey_count+count)
```

Comment:
If I understand correctly, you're correcting an unrelated bug here (but I'm a bit lost).
Isn't the problem inside kraken, more?
Why add instead of concatenate?

Reply:
yes, accepted